index.rst

```diff
@@ -48,10 +48,63 @@
 
    **This technote is not yet published.**
 
-   This proposal outlines how observing tasks will be flowed from the planning stage through execution, then linked to data analysis technotes and/or notebooks.
+   This proposal outlines how observing tasks will be flowed from the planning stage through execution,
+   then linked to data analysis technotes and/or notebooks.
```

Comment:
Using Jira as a central workflow management tool, similar to what is done in the LVV project, is key to linking the pieces together, monitoring the process, and making it visible to all the stakeholders..